src/lib/core/CHIPError.h

```diff
@@ -2372,6 +2372,14 @@ using CHIP_ERROR = ::chip::ChipError;
  */
 #define CHIP_ERROR_BUSY                     CHIP_CORE_ERROR(0xdb)
 
+/**
+ * @def CHIP_ERROR_MAX_RETRY_EXCEEDED
+ *
+ * @brief
+ *   The Resource has exceeded the maximum rety limit.
```

Comment:
What is "The Resource"? Also, typo on "retry". Is it sufficient to just say?
```suggestion
 *   The maximum retry limit has been exceeded.
```

Reply:
Made the suggested comment changed.